advancedlog/types.go

```diff
@@ -0,0 +1,29 @@
+package logger
+
+// Producer ...
+type Producer string
+
+const (
+	// CLI ...
+	CLI Producer = "cli"
```

Comment:
IMO we shouldn't use `cli` as it is too generic, let's use `bitrise cli`

Reply:
Yes, definitely but I will add an underscore (`bitrise_cli`) to it because all of our multi word values have an underscore in them.